Shared/Samples/Change map view background/ChangeMapViewBackgroundView.Model.swift

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import SwiftUI
+import ArcGIS
+
+extension ChangeMapViewBackgroundView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A map with a single feature layer representing world time zones.
+        let map: Map = {
+            // Initialize tiled layer.
+            let tiledLayer = ArcGISTiledLayer(url: .worldTimeZones)
+            
+            // Initialize map with tiled layer as basemap.
```

Comment:
```suggestion
            // Initializes map with tiled layer as basemap.
```

Reply:
Or "Initializes a map with tiled layer as the basemap."